waywiser_teleop/py_scripts/twist_keyboard.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
```

Comment:
That this is python makes me suspicious. 😄 Is this based on something existing that we should to reference?

Reply:
It was a bit surprising that I couldn't find any standard cpp library to record multiple key presses, but it works like a charm with pygame python module. It wasn't based on any existing work.